examples/redux/src/redux/modules/commits.js

```diff
@@ -0,0 +1,44 @@
+import * as Repo from './repos';
+import * as Users from './users';
+import { normalize } from '../../../../../src';
+
+export const STATE_KEY = 'commits';
+
+export default function reducer(state = {}, action) {
+  switch (action.type) {
+    case Action.ADD_COMMITS:
+      return {
+        ...state,
+        ...action.payload
+      };
+
+    default:
+      return state;
+  }
+}
+
+const Action = {
+  ADD_COMMITS: 'ADD_COMMITS'
+};
+
+export const addCommits = (commits = {}) => ({
+  type: Action.ADD_COMMITS,
+  payload: commits
+});
+
+export const getCommits = ({ page = 0 } = {}) => (dispatch, getState, { api, schema }) => {
+  const state = getState();
+  const owner = Repo.selectOwner(state);
+  const repo = Repo.selectRepo(state);
+  return api.repos.getCommits({
+    owner,
+    repo
+  }).then((response) => {
+    const data = normalize(response, [ schema.commit ]);
+    dispatch(Users.addUsers(data.entities.users));
+    dispatch(addCommits(data.entities.commits));
```

Comment:
I agree. We use a batching middleware in mobile.twitter.com, but I was hoping to keep the normalizr example clear without using too much redux middleware. I cringed a little adding the thunk, even. Also, the reason to not dispatch multiple is typically React-related, since you don't want to trigger multiple render cycles. Since this doesn't include react and it's just an example, maybe a comment about best practices inline would be better?